src/openforms/payments/contrib/ogone/tests/test_client.py

```diff
@@ -1,14 +1,62 @@
+import os
+from pathlib import Path
+
 from django.test import TestCase
 
+import requests
+
+from openforms.utils.tests.vcr import OFVCRMixin
+
 from ..client import OgoneClient
 from ..plugin import RETURN_ACTION_PARAM
 from .factories import OgoneMerchantFactory
 
 
-class OgoneClientTest(TestCase):
+class OgoneClientTest(OFVCRMixin, TestCase):
```

Comment:
In addition to the Taiga documentation, can you also update the [release issue template](https://github.com/open-formulieren/open-forms/blob/master/.github/ISSUE_TEMPLATE/prepare-release.md)?